### PR TITLE
Acceptance test lighten

### PR DESCRIPTION
**Pull request by flynnbops**
Reduced the number of scenarios (and therefore signings) in the Acceptance tests.

### CODE REVIEW

File: features/borrowers_view_and_sign_deed.feature [hunk 1]
```diff
@@ -18,255 +18,67 @@ Scenario: Single borrower views and signs deed, then views
...
   And borrower <1> views the deed
+  And the mortgage deed is displayed
   Then I verify borrower <1> has signed the deed
 
 @us133 @us133f
-Scenario: Single borrower views deed, then views and signs
-  Given I create default deed with <1> borrowers
-  And borrower <1> views the deed
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
```
Comment (nigelpain) [inline features/borrowers_view_and_sign_deed.feature]: Does this need to be removed, as this test isn't actually signing the deed?

File: features/borrowers_view_and_sign_deed.feature [hunk 2]
```diff
@@ -18,255 +18,67 @@ Scenario: Single borrower views and signs deed, then views
...
-
-@us133 @us133f
-Scenario: Single borrower views deed 3 times
-  Given I create default deed with <1> borrowers
-  And borrower <1> views the deed
-  And borrower <1> views the deed
-  Then borrower <1> views the deed
```
Comment (nigelpain) [inline features/borrowers_view_and_sign_deed.feature:35]: Does this need to be removed, as this test isn't actually signing the deed?

File: features/borrowers_view_and_sign_deed.feature [hunk 3]
```diff
@@ -18,255 +18,67 @@ Scenario: Single borrower views and signs deed, then views
...
+  And the mortgage deed is displayed
   Then I verify borrower <1> has signed the deed
-  Then I verify borrower <2> has signed the deed
-
-@us133 @us133f
-Scenario: 3 Borrowers can view a deed. p1
-  Given I create default deed with <3> borrowers
-  And borrower <2> views the deed
-  And borrower <1> views the deed
-  Then borrower <3> views the deed
```
Comment (nigelpain) [inline features/borrowers_view_and_sign_deed.feature:104]: Same comment as above regarding does this need to be removed as no signing is taking place?

File: features/borrowers_view_and_sign_deed.feature (hunk 2 above)
Comment (flynnbops) [inline features/borrowers_view_and_sign_deed.feature:35]: The step "Then the deed is digitally signed by borrower <1>" performs the signing process.

A borrower viewing the deed seemed a bit redundant, I'm happy to put it back in.

File: features/borrowers_view_and_sign_deed.feature [hunk 4]
```diff
@@ -18,255 +18,67 @@ Scenario: Single borrower views and signs deed, then views
...
-  Then I verify borrower <2> has signed the deed
-  Then I verify borrower <3> has signed the deed
-
-@us133 @us133f
-Scenario: 4 Borrowers can view a deed
-  Given I create default deed with <4> borrowers
-  And borrower <2> views the deed
-  And borrower <4> views the deed
-  And borrower <1> views the deed
-  Then borrower <3> views the deed
```
Comment (nigelpain) [inline features/borrowers_view_and_sign_deed.feature:178]: Again same as two comments above regarding keeping this in.

File: features/borrower_requests_auth_code.feature [hunk 5]
```diff
@@ -12,21 +12,3 @@ Scenario: Sign my mortgage page tells borrower they need their mobile phone
...
-@us46
-Scenario: 2 Borrowers Request Auth Code to sign deed
-  Given I create default deed with <2> borrowers
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
-  And borrower <2> views the deed
-  And I verify borrower <1> has signed the deed
-  Then the deed is digitally signed by borrower <2>
-  And borrower <2> views the deed
-  And I verify borrower <2> has signed the deed
```
Comment (nigelpain) [inline features/borrower_requests_auth_code.feature:32]: I would want to keep one of these scenarios in as unless I've missed something I don't think this requesting the code bit is checked anywhere else?

**Comment by nigelpain** [comment]
Few comments around leaving some of the tests in.

File: features/borrower_requests_auth_code.feature (hunk 5 above)
Comment (flynnbops) [inline features/borrower_requests_auth_code.feature:32]: As discussed this is now part of the overall flow. So these tests are redundant

**Comment by nigelpain** [comment]
:+1: with the view deed tests back in

**Pull request merged**